tests/echo_module_test.cc

```diff
@@ -27,7 +27,7 @@ TEST(EchoModuleTest, createEchoModule) {
             {"path", "/foo"},
             });
 
-    std::unique_ptr<Module> mod(EchoModule::createFromParameters(paramMap));
+    std::unique_ptr<Module> mod(EchoModule::createFromParameters("/foo", paramMap));
```

Comment:
If you do `using namespace std;`, it will pull *all* of `std` into the global namespace. So if we happen to have name collisions, it becomes unclear what we mean. It's better if we just selectively do `using std::string;` for each thing we use. In this case, I think we only use `unique_ptr` a couple of times so I didn't bother with `using std::unique_ptr;`.